app/models/member.rb

```diff
@@ -145,6 +145,25 @@ def access_level_in_namespace_group_links(user, namespace)
          Member.effective_access_level(effective_namespace_group_link&.group, user)].min
       end
     end
+
+    def user_emails(namespace)
+      user_memberships = Member.for_namespace_and_ancestors(namespace).not_expired
+                               .where.not(access_level: Member::AccessLevel.manageable)
+      users = User.human_users.where(id: user_memberships.select(:user_id)).distinct
+      users.pluck(:email)
+    end
```

Comment:
No longer need to remove managers from `user_memberships`

Reply:
afc64834a297ec27332b860346f0d0e66c2b927b